go-controller/pkg/ovn/gress_policy.go

```diff
@@ -244,116 +262,115 @@ func addACLAllow(np *namespacePolicy, match, l4Match string, ipBlockCidr bool, g
 		fmt.Sprintf("action=%s", action),
 		fmt.Sprintf("external-ids:l4Match=\"%s\"", l4Match),
 		fmt.Sprintf("external-ids:ipblock_cidr=%t", ipBlockCidr),
-		fmt.Sprintf("external-ids:namespace=%s", np.namespace),
-		fmt.Sprintf("external-ids:policy=%s", np.name),
-		fmt.Sprintf("external-ids:%s_num=%d", policyType, gressNum),
-		fmt.Sprintf("external-ids:policy_type=%s", policyType),
-		"--", "add", "port_group", np.portGroupUUID, "acls", "@acl")
+		fmt.Sprintf("external-ids:namespace=%s", gp.namespace),
+		fmt.Sprintf("external-ids:policy=%s", gp.name),
+		fmt.Sprintf("external-ids:%s_num=%d", gp.policyType, gp.idx),
+		fmt.Sprintf("external-ids:policy_type=%s", gp.policyType),
+		"--", "add", "port_group", gp.portGroupUUID, "acls", "@acl")
```

Comment:
It would work fine to pass `portGroupName` here instead of `portGroupUUID` wouldn't it? And then you don't need `portGroupUUID`.

Reply:
@danwinship I believe that's a database command and the port_group table is indexed on UUID so I don't think that's possible right now.